src/server/server.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import { createServer } from 'http';
+
+import { MIME_TYPES, STATUS_CODES, RPC_DEFAULT_PORT } from '../defaults';
+
+import type { ServerArgumentsType } from '../flowtypes';
+
+export const start = async (
+  { port = RPC_DEFAULT_PORT }: ServerArgumentsType = {},
```

Comment:
This has no re-use value and should be defined locally as opposed to being a named type inside the `flowtypes` file.

Eg:
```js
export const start = async (
  { port = RPC_DEFAULT_PORT }: { port: number } = {},
): Promise<Object> => {
  ...
};
```